### DISCUSSION TITLE
compatibility with kineticist expanded elements

### DISCUSSION THREAD

**Discussion by decourcy2** [Truinto/DarkCodex#111] [Q&A] [answered]
Hi, I am using both mods, hit lvl 7 with my water kineticist, chose air. I now have water blast and electric blast which should give charged water blast. I did not get any composite blast. 
It seems others are not having this problem, is this a compatibility issue?
Thanks for your time.

**Comment by Truinto** [comment, marked as answer]
At first I could reproduce it. But apparently the issue resolve itself after saving and reloading.

**Reply by decourcy2** [reply]
Thank you, I cannot believe (figuratively) turning my computer off and on again fixed this, that is a new weird thing Owlcat.
